r2snowman/r2snow.pl

```diff
@@ -1,62 +0,0 @@
-#!/usr/bin/env perl
-# generate r2 script out of a snowman decompilation
-# author : pancake // nopcode.org // 2016
-
-sub rf {
-  my ($file) = shift;
-  open FH, $file or die "cannot open $file";
-  my @lines = <FH>;
-  close FH;
-  return @lines;
-}
-
-sub wf {
-  my ($file, $data) = @_;
-  open FH, ">$file" or die "cannot open $file";
-  print FH $data;
-  close FH;
-}
-
-use constant {
-  R2SNOW_SOURCE => "r2snow-source.c"
-};
-
-my $source = join("", rf(R2SNOW_SOURCE));
-my @addrof = rf("r2snow-addrof.txt");
-
-my $ts = $source;
-$ts =~s/reinterpret_cast//g;
-$ts =~s/static_cast//g;
-wf(R2SNOW_SOURCE . ".txt", $ts);
-
-local @nls = ();
-sub initNewlines() {
-  my $offset = 0;
-  my $result = index($source, "\n", $offset);
-  while ($result != -1) {
-    push (@nls, $offset);
-    $offset = $result + 1;
-    $result = index($source, "\n", $offset);
-  }
-}
-
-sub lineFor($) {
-  my ($offset) = @_;
-  my $line = 0;
-  foreach my $nl (@nls) {
-    if ($nl >= $offset) {
-      return $line;
-    }
-    $line++;
-  }
-  return 0;
-}
-
-initNewlines();
-
-foreach my $a (@addrof) {
-  my ($addr, $offset) = split(" ", $a);
-  my $file = R2SNOW_SOURCE . ".txt";
-  my $line = lineFor($offset);
-  print "CL $file:$line $addr\n";
```

Comment:
you are also removing the ability to import the decompilation into r2

Reply:
Hmm. I noticed that previously r2 used "feature/output-ranges" branch of snowman. But just-function decompilation implemented only in main branch. So I need to make one more pool request to this branch